packages/@okta/vuepress-site/docs/reference/api/factors/index.md

```diff
@@ -3142,6 +3390,131 @@ curl -v -X POST \
 }
 ```
 
+### Verify WebAuthn Factor
+
+<ApiOperation method="post" url="/api/v1/users/${userId}/factors/${factorId}/verify" />
+
+Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options, which are used to help select an appropriate authenticator using the WebAuthn API.
```

Comment:
that rather than which here:

Verifies a user with a WebAuthn factor. The verification process starts with getting the WebAuthn credential request options that are used to help select an appropriate authenticator using the WebAuthn API.